controllers/cache_handler.go

```diff
@@ -243,7 +243,11 @@ func (h DisruptionTargetWatcherHandler) findNotifiableEvents(eventsToSend map[st
 					"message", event.Message,
 					"timestamp", event.LastTimestamp.Time.Unix(),
 				)
-			} else if event.Reason == "Started" {
+
+				continue
```

Comment:
what's the difference between continuing here vs. using `else if` as before?

Reply:
the linter was complaining as it was making a forest of ifs 😞

Reply:
I think `switch` might be more idiomatic in Go then? I'd rather suppress the linter than use the continue here, at least personally